### DISCUSSION TITLE
Not able to open Jupiter in Docker(linux)

### DISCUSSION THREAD

**Discussion by sainadh812** [Xilinx/finn#986] [Q&A] [answered]
![image](https://github.com/Xilinx/finn/assets/58074126/6717282f-a946-4989-b7f3-d40872de7c7f)

it is showing like above image at end when i try to run "bash ./run-docker.sh notebook"

![image](https://github.com/Xilinx/finn/assets/58074126/3c46336b-e4de-4e7a-b52c-ec64e2f0be6c)

i need help to resolve these issues.

**Comment by fpjentzsch** [comment, marked as answer]
Should be fixed on dev branch, see https://github.com/Xilinx/finn/discussions/889

**Comment by sainadh812** [comment]
![image](https://github.com/Xilinx/finn/assets/58074126/9028ed51-a000-435a-b698-8b26ab11429a)

i clone the finn DEV branch using "git clone --branch dev https://github.com/Xilinx/finn.git"

and that i run "bash ./run-docker.sh notebook"

while running this , installation is getting stuck as shown in above image above

**Reply by auphelia** [reply]
See reply in the other Discussion you opened: https://github.com/Xilinx/finn/discussions/988
